guides/common/modules/proc_adding-lifecycle-environments.adoc

```diff
@@ -69,3 +69,12 @@ Repeat for each lifecycle environment you want to add to {SmartProxyServer}.
 --id _My_{smart-proxy-context}_ID_
 --lifecycle-environment-id _My_Lifecycle_Environment_ID_
 ----
++
+* To synchronize all content from your {ProjectServer} to your {SmartProxyServer} without skipping checks on metadata:
```

Comment:
I think it would be worthwhile to mention that "without skipping checks on metadata" is also called a "complete sync" in our UI.

Reply:
We _are_ skipping the checks, though?

```
--skip-metadata-check true
```

Reply:
Yes, Quirin found the same issue with my wording :see_no_evil: We've already fixed it: https://github.com/theforeman/foreman-documentation/pull/2674/files